handbook/company/README.md

```diff
@@ -29,7 +29,7 @@ Fleet is currently hiring for the following positions:
 
 - [🐋 Head of Public Sector](https://fleetdm.com/handbook/company/head-of-public-sector)
 - [🐋 Solutions Consultant](https://fleetdm.com/handbook/company/solutions-consultant)
-- [🐋 Customer Support Engineer](https://fleetdm.com/handbook/company/customer-support-engineer)
+- [🐋 Customer Support Engineer]
```

Comment:
I think this should either be `🐋 Customer Support Engineer` (without the square brackets) -or- a change to the linked page to remove the Typeform link and suggest reaching out on linkedin instead.  (My guess is that we're getting overwhelmed with more applicants than we can handle again due to some kind of automation adding this automatically to job lists)